src/api/list_add.sql

```diff
@@ -0,0 +1,75 @@
+/*
+ * Function api.list_add adds one element or several elements to the list.
+ * Input required parameter is jsonb with:
+ * class - the class of the object
+ * id - id of the object 
+ * field - the name of the field to add the value to
+ * value - one scalar value or array of values
+ * access_token - jwt token to authorize
+*/
+
+CREATE OR REPLACE FUNCTION api.list_add(data jsonb)
+RETURNS void AS $$
+DECLARE
+    class          jsonb;
+    obj_id         uuid;
+    values_to_add  jsonb;
+    json_path      text[];
+    obj            jsonb;
+    new_obj        jsonb;
+    field_value    jsonb;
+    access_token   jsonb;
+
+begin
+    class := data->'class';
+    IF (class is null) THEN
```

Comment:
Both "IS" and "is", "NULL" and "null" are currently used in the code. I suggest that we use a single notation through the code; "IS" and "NULL" will do, as caps better distinguish SQL keywords from variables etc.